clang-tools-extra/clangd/tool/Check.cpp

```diff
@@ -163,7 +163,7 @@ class Checker {
   unsigned ErrCount = 0;
 
   Checker(llvm::StringRef File, const ClangdLSPServer::Options &Opts)
-      : File(File), Opts(Opts) {}
+      : File(File), Opts(Opts), Index(/*SupportContainedRefs=*/false) {}
```

Comment:
why false here? i think it's better if we use whatever clangd-server's default is

Reply:
My original thinking was that `--check` does not perform any outgoing calls requests. But thinking more about it, a common use of `--check` is to reproduce crashes, so the more codepaths it executes (including the one behind this flag) the better in that regard.